pkg/core/xds/testdata/rules/02.golden.yaml

```diff
@@ -0,0 +1,103 @@
+- Conf:
+    action: ALLOW
+  Origin: null
+  Subset:
+  - Key: env
+    Not: false
+    Value: dev
+  - Key: version
+    Not: false
+    Value: v1
+  - Key: zone
+    Not: false
+    Value: us-east
+- Conf:
+    action: DENY
+  Origin: null
+  Subset:
+  - Key: env
+    Not: true
+    Value: dev
+  - Key: version
+    Not: false
+    Value: v1
+  - Key: zone
+    Not: false
+    Value: us-east
+- Conf:
+    action: ALLOW
+  Origin: null
+  Subset:
+  - Key: env
+    Not: false
+    Value: dev
+  - Key: version
+    Not: false
+    Value: v1
+  - Key: zone
+    Not: true
+    Value: us-east
+- Conf:
+    action: ALLOW
+  Origin: null
+  Subset:
+  - Key: env
+    Not: false
+    Value: dev
+  - Key: version
+    Not: true
+    Value: v1
+  - Key: zone
+    Not: false
+    Value: us-east
+- Conf:
+    action: DENY
+  Origin: null
+  Subset:
+  - Key: env
+    Not: true
+    Value: dev
+  - Key: version
+    Not: true
+    Value: v1
+  - Key: zone
+    Not: false
+    Value: us-east
+- Conf:
+    action: ALLOW
+  Origin: null
+  Subset:
+  - Key: env
+    Not: true
+    Value: dev
+  - Key: version
+    Not: false
+    Value: v1
+  - Key: zone
+    Not: true
+    Value: us-east
+- Conf:
+    action: ALLOW
+  Origin: null
+  Subset:
+  - Key: env
+    Not: false
+    Value: dev
+  - Key: version
+    Not: true
+    Value: v1
+  - Key: zone
+    Not: true
+    Value: us-east
+- Conf: null
+  Origin: null
```

Comment:
Can we have `omitempty` on the object?

Reply:
We can, but why do we want that? This structure is marshaled only for golden-test purposes, it's not exposed to API